src/shared/datePicker.js

```diff
@@ -8,12 +9,12 @@ const fixDate = (dateValue) => {
   if (!dateValue) {
     return '';
   }
-  const date = dateValue.split('T')[0];
+  const date = dateValue.format('YYYY-MM-DDTHH:mm:ss').split('T')[0];
```

Comment:
this is now definitely the same logic as localToISOString (aka formatISODate). Remove the whole fixDate thingy and call formatISODate. Alternatively if need be put a guard in formatISODate that checks if dateValue is set